v0/destinations/marketo/util.js

```diff
@@ -30,45 +33,45 @@ const logger = require("../../../logger");
 const marketoApplicationErrorHandler = (
   marketoResponse,
   sourceMessage,
-  stage
+  destination
```

Comment:
We have omitted stage how are we going to know where this failures are happening is it at proxy or destination transformation.

Reply:
We have to pass the stage information when we are throwing errors

Reply:
I saw in marketo that it is passed as hardcoded whenever we are throwing an error and not using stage one. 
Don't know why